### PR TITLE
feat : added validation in sign up form

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added sign up page with Validation

### CODE REVIEW

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    
+    email.value === '' || email.value == null ? valEmail.innerText = "Please enter valid email id." : valEmail.innerText = "";
+
+    password.value.length < 8 || password.value.length > 15 || password.value === '' || password.value == null ? valPassword.innerText = "Password must contain 8-15 characters"  : valPassword.innerHTML = "";
+    
+    username.value === '' || username.value == null ? username.classList.add("input-danger","border-danger","border-success","input-success") : username.classList.remove("input-danger","border-danger");
+
+    email.value === '' || email.value == null ? email.classList.add("input-danger","border-danger","border-success","input-success") : email.classList.remove("input-danger","border-danger");
+
+    password.value === '' || password.value == null ? password.classList.add("input-danger","border-danger","border-success","input-success") : password.classList.remove("input-danger","border-danger");
```
Comment (barbaricankit) [inline validation.js]: you could have used if else, if have same conditions in two places, only the statement is changing.

File: validation.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const form = document.getElementById("form");
+const username = document.getElementById("username");
+const email = document.getElementById("email");
+const password = document.getElementById("password");
+const btnSubmit = document.getElementById("btn-submit");
+const btnCancel = document.getElementById("btn-cancel");
+const valUserName = document.getElementById("valid-username");
+const valEmail = document.getElementById("valid-email");
+const valPassword = document.getElementById("valid-password");
```
Comment (barbaricankit) [inline validation.js]: You should have used invalid prefix as those texts are for invalid situations

**Review by rkapoor10 — APPROVED** [review]
Just a suggestion - you may show validation on a button click instead of just filling one field. Otherwise looks good to merge.

**Review by barbaricankit — COMMENTED** [review]
You have not given any PR description, you should always give that.
I have provided some comments, please check those.

File: validation.js (hunk 2 above)
Comment (Omkar-Chaskar) [reply validation.js]: Thanks . i will change id name to invalid .

**Pull request merged**